rotkehlchen/tasks/manager.py

```diff
@@ -831,6 +831,13 @@ def _maybe_create_calendar_reminder(self) -> Optional[list[gevent.Greenlet]]:
         ):
             return None
 
+        with self.database.conn.write_ctx() as write_cursor:
+            self.database.set_static_cache(  # remember last task ran
+                write_cursor=write_cursor,
+                name=DBCacheStatic.LAST_CREATE_REMINDER_CHECK_TS,
+                value=ts_now(),
+            )
```

Comment:
This should be done in the task itself. The reason is that if for some reason the task dies before executing (say that you close the app) then it won't have executed but the timestamp would have been recorded

```suggestion
            self.database.set_static_cache(
                write_cursor=write_cursor,
                name=DBCacheStatic.LAST_CREATE_REMINDER_CHECK_TS,
                value=ts_now(),
            )
```